### ISSUE TITLE
Support RxJava3 

### ISSUE THREAD

**Issue by WhosNickDoglio** [square/workflow-kotlin#524]
There's an RxJava2 artifact are there plans for an RxJava3 artifact? 

Would you be happy to open a PR for it you're open to it

**Comment by rjrjr** [comment]
Thanks for the offer, but I don't think we want to imply that we have the bandwidth to support RxJava3. 

The good news is that it should be straightforward to build such support without needing any changes in this repo, or access to any private APIs.

**Comment by rjrjr** [comment]
cc @steve-the-edwards, @zach-klippenstein for other perspectives.

**Comment by zach-klippenstein** [comment]
No, I don't think it makes sense for us to maintain that. Rx2 is basically just there to support migrations away from it anyway.
